shell/plugins/dashboard-store/actions.js

```diff
@@ -440,7 +441,7 @@ export default {
       const schema = getters['schemaFor'](type);
 
       if (schema.attributes?.namespaced) {
-        const [namespace, realId] = id.split('/');
+        const [namespace, realId] = id.split('/'); // TODO: RC id is namespace/id... which is the resource id. why need just the id??
```

Comment:
Because we're eventually sending this to a generic function that'll process id, namespace, and type altogether it's simpler to break them into separate parameters here.

Consider, valid requests could be:
type (findAll)
type/id (find)
type/namespace (findAll)
type/namespace/id (find)

so it makes sense for the request being sent to the worker to always know specifically what the namespace is even if we've glommed the namespace and the id into a single string when we fed it into the vuex action. In a perfect world, we'd trigger the action with both the namespace and the id as separate parameters but that'd be overstepping our directive with what we're doing now so I'm just accounting for it here.

Reply:
have moved this into the steve request part